app/ide-desktop/eslint.config.js

```diff
@@ -0,0 +1,354 @@
+/** @file ESLint configuration file. */
+/** NOTE: The "Experimental: Use Flat Config" option must be enabled. */
+import * as path from 'node:path'
+import * as url from 'node:url'
+
+const DIR_NAME = path.dirname(url.fileURLToPath(import.meta.url))
+
+// The .d.ts is wrong; these only have default exports.
+// Other default imports are whitelisted; however these should only be used here.
+/* eslint-disable no-restricted-syntax */
+import eslintJs from '@eslint/js'
+import globals from 'globals'
+import jsdoc from 'eslint-plugin-jsdoc'
+import tsEslint from '@typescript-eslint/eslint-plugin'
+import tsEslintParser from '@typescript-eslint/parser'
+/* eslint-enable no-restricted-syntax */
```

Comment:
Why we need this?

Reply:
this one is needed to re-enable the lint disabled on line 10, otherwise the rest of the document won't be linted